desktop/renderer/index.js

```diff
@@ -77,16 +79,31 @@ function setupApp (store) {
   })
 
   const currentWindow = electron.remote.getCurrentWindow()
-  currentWindow.on('focus', () => {
-    store.dispatch(changedFocus(true))
-  })
-  currentWindow.on('blur', () => {
-    store.dispatch(changedFocus(false))
-  })
+  // This fixes reload problems with stale listeners
```

Comment:
i'm talking about reloading the window with command+r. if you do that your old eventHandlers still exist but in a busted state.  we were suffering from this: https://github.com/electron/electron/issues/3778